data/tlg0007/tlg068/tlg0007.tlg068.perseus-grc2.xml

```diff
@@ -0,0 +1,2231 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="http://www.stoa.org/epidoc/schema/latest/tei-epidoc.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0">
+<teiHeader xml:lang="eng">
+<fileDesc>
+<titleStmt>
+<title xml:lang="lat">Quomodo adolescens poetas audire debeat</title>
+<!--<title type="sub">Machine readable text</title>-->
+<author>Plutarch</author>
+<editor role="editor" n="Teubner">Gregorius N.
+            Bernardakis</editor>
+<sponsor>Perseus Project, Tufts University</sponsor>
+<principal>Gregory Crane</principal>
+<respStmt>
+<resp>Prepared under the supervision of</resp>
+<name>Lisa Cerrato</name>
+<name>William Merrill</name>
```

Comment:
Please remove the names Merrill, Smith and Mylonas. Please add the names Rashmi Singhal and Bridget Almas. (This is something I need to edit and saves me a PR.) Thanks!